Dockerfile

```diff
@@ -18,6 +18,9 @@ COPY app/console /usr/local/src/console
 
 WORKDIR /usr/local/src/console
 
+ARG VITE_GA_PROJECT=""
+ENV VITE_GA_PROJECT=$VITE_GA_PROJECT
```

Comment:
This also needs to be added to the `docker-compose.yml` file for the appwrite service.

Reply:
Why though? Isn't it only required during build time ? ###